numpy/lib/function_base.py

```diff
@@ -1921,36 +1951,77 @@ def cov(m, y=None, rowvar=1, bias=0, ddof=None):
         y = np.asarray(y)
         dtype = np.result_type(m, y, np.float64)
     X = array(m, ndmin=2, dtype=dtype)
+    if rowvar == 0 and X.shape[0] != 1:
```

Comment:
Hmm, I see this is inherited, `X.shape[0] == 1]` always overrode `rowvar == 0`. Doesn't seem right, but that is how it was...